### PR TITLE
[3.4] drop collection action timeout

### PR DESCRIPTION
**Pull request by kvahed**

### CODE REVIEW

**Comment by neunhoef** [comment]
This can be merged and the fix should still go into 3.4.8.

**Comment by neunhoef** [comment]
Jenkins was fully green on it: http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/6022/

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,8 @@
 v3.4.8 (2019-09-XX)
 -------------------
 
+* Drop collection action to timeout more quickly to stay on fast lane
```
Comment (joerg84) [inline CHANGELOG]: Decreased timeout for drop collection action to stay on fast lane.

**Comment by kvahed** [comment]
Sure, will change, but it was completely perfect and arguably better English

File: CHANGELOG (hunk 1 above)
Comment (kvahed) [reply CHANGELOG]: Sure, would change, if you insist, but the current version is perfect and arguably better English

**Comment by joerg84** [comment]
> Sure, will change, but it was completely perfect and arguably better English
> […](#)
> On 4. Sep 2019, at 11:21, Joerg Schad ***@***.***> wrote: @joerg84 commented on this pull request. In CHANGELOG: > @@ -1,6 +1,8 @@ v3.4.8 (2019-09-XX) ------------------- +* Drop collection action to timeout more quickly to stay on fast lane Decreased timeout for drop collection action to stay on fast lane. — You are receiving this because you were assigned. Reply to this email directly, view it on GitHub, or mute the thread.

Just tried to match the consistent format of the changelog. (i.e., past tense and `.`. at the end of the sentence.

**Pull request merged**